apps/condo/domains/common/utils/sberCloudFileAdapter.js

```diff
@@ -0,0 +1,78 @@
+const ObsClient = require('esdk-obs-nodejs')
+class SberCloudFileAdapter {
+
+    constructor (config) {
+        this.bucket = config.bucket
+        if (!this.bucket) {
+            throw new Error('SberCloudAdapter: requires a bucket name')
+        }
+        this.s3 = new ObsClient(config.s3Options)
```

Comment:
move obs to constructor args

Reply:
this is probably an internal implementation detail. And perhaps it is not necessary to put it inside an arguments

Reply:
It's much easier to test this abstraction when client is isolated to constructor arguments. And this is common patter of dependency inversion, it's not about implementation